src/_support/Page/Acceptance/Administrator/MenuItemFormPage.php

```diff
@@ -0,0 +1,152 @@
+<?php
+/**
+ * @package  Joomla.Test
+ * @subpackage  AcceptanceTester.Page
+ *
+ * @copyright   Copyright (C) 2005 - 2018 Open Source Matters, Inc.All rights reserved.
+ * @license     GNU General Public License version 2 or later;see LICENSE.txt
+ *  7.0.30
+ */
+namespace Page\Acceptance\Administrator;
+
+/**
+ * Acceptance Page object class to menuitem form page.
+ *
+ * @category  Users
+ * @package   Page\Acceptance\Administrator
+ * @copyright 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license   GNU General Public License version 2 or later; see LICENSE.txt
+ * @since     4.0.0
+ */
+class MenuItemFormPage
+{
+	/**
+	 * Menu Item Title
+	 *
+	 * @var   string
+	 * @since 4.0.0
+	 */
+	public static $menuItemTitle = '#jform_title';
+
+	/**
+	 * Menu Item Alias
+	 *
+	 * @var   string
+	 * @since 4.0.0
+	 */
+	public static $menuItemAlias = '#jform_alias';
+
+	/**
+	 * Menu Item Save
+	 *
+	 * @var   string
+	 * @since 4.0.0
+	 */
+	public static  $saveButton = '#toolbar-apply';
+
+	/**
+	 * Page title of the user manager listing page.
+	 *
+	 * @var   string
+	 * @since 4.0.0
+	 */
+	public static $pageTitleText = "Menus";
+
+	/**
+	 * A drop down menu
+	 *
+	 * @var   array
+	 * @since 4.0.0
+	 */
+	public static $menuDropDown = ['xpath' => '//select[@id="jform_menutype"]'];
```

Comment:
change in locators as I mentioned, we are not going to use arrays any more, lets move this one to a string like this: 
`"//select[@id="jform_menutype"]`